test/CI/Travis/apache_vhost

```diff
@@ -0,0 +1,68 @@
+# Configuration file for Apache running on Travis.
```

Comment:
Not really, I just took a vhost I had used on Travis for another project (you can see I even forgot to remove the php part).

The custom vhost gives us more room to expand in the future, but I can go for the shorter version if preferred